workstation/linux/README.md

```diff
@@ -0,0 +1,79 @@
+# How to compile NEQUIP + AiiDA + SSCHA in a container/fresh computer
+
+Using a MAMBA (a faster version of conda) environment is the easiest way.
+
+These instructions should work out of the box. If you encounter any issue, you can:
+1. See _Known issues_ at the bottom.
+2. Visit the other compilation instructions
+3. Contact me.
+
+
+## Installation instruction
+
+Create the conda environment:
+```console
+mamba create -yn sscha-nequip -c conda-forge python=3.9 git \
+spglib=2.2 scipy pytest julia gfortran libblas lapack pip gcc gxx \
+cmake openmp liblapacke openblas aiida-core aiida-core.services
+mamba activate sscha-nequip
+```
+
+Clone the following packages under development:
+```console
+git clone -b new/flare-interface --depth 1 https://github.com/bastonero/python-sscha.git
+git clone --depth 1 https://github.com/SSCHAcode/CellConstructor.git
+```
+
+Install Pytorch (needed for NequIP):
+```console
+conda install -yn sscha-nequip -c conda-forge pytorch~=1.12.1 pytorch-cpu~=1.12.1
+```
+
+Install aiida-quantumespresso, NequIP and some other packages via pip:
+```console
+pip install ase==3.22 julia aiida-quantumespresso aiida-pseudo nequip
+```
+
+One should also consider properly installing AiiDA, making an AiiDA profile afterwards, and setting computer and codes.
+
+
+Install the SSCHA and its dependencies:
+```console
+cd CellConstructor
+python setup.py install
+cd ../python-sscha
+python setup.py install
+pip install tdscha
+```
+
+Finally, the SSCHA benefits from julia to automatically speed up the calculation. Install the julia dependencies:
+```console
+python -c 'import julia; julia.install()'
+```
+
+Reinstall numpy enabling the OMP paralelization:
+```console
+pip install --no-binary :all: numpy~=1.22.0 --force-reinstall
+```
```

Comment:
This is only needed for reinstalling numpy and linking it to available linear algebra packages, such as intel MKL. But here it is not necessary, there is already openblas and liblapacke in the conda environment. So I would drop these lines